queue_processors/autoreduction_processor/tests/test_post_process_admin.py

```diff
@@ -111,6 +111,30 @@ def test_load_reduction_script(self):
         self.assertEqual(file_path, os.path.join(MISC['scripts_directory'] % 'WISH',
                                                  'reduce.py'))
 
+    def test_specify_instrument_directories(self):
+        """
+        Test: Expected instrument, stripped of run number if excitation returned
+        When: called
+        """
+        ppa = PostProcessAdmin(self.message, None)
+        ppa.instrument = 'WISH'
+        ppa.run_number = 27282
+        actual = ppa.specify_instrument_directories(
+            instrument_output_directory=MISC["ceph_directory"] % (ppa.instrument,
+                                                                  ppa.proposal,
+                                                                  ppa.run_number),
+            excitations=MISC["excitation_instruments"],
+            temporary_directory=MISC["temp_root_directory"])
+
+        directory_list = [i for i in actual.split('/') if i]
+
+        expected_end = ['reduced-data', 'WISH', 'RB1234', 'autoreduced', '27282']
+        expected_start = 'autoreducetmp'
+
+        self.assertEqual(directory_list[-5:], expected_end)
+        self.assertEqual(directory_list[0], expected_start)
+        self.assertIsInstance(actual, str)
```

Comment:
We don't need this last assert since `assertEqual(2782, "2782")` is not true, so the fact our expected is already a string means this is guaranteed to be a string too

Reply:
Also were hard coding `reduced-data` and `autoreduced` which makes for a brittle test, since these can change elsewhere causing an unrelated failure here for someone to come in and fix later. 
Is there a way to just get the values from `MISC` here rather than hard coding them?